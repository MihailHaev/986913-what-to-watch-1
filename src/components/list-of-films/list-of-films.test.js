import React from "react";
import renderer from "react-test-renderer";
import ListOfFilms from "./list-of-films";

const mock = {
  films: [{
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    src: ``,
  }, {
    title: `Bohemian Rhapsody`,
    img: `img/bohemian-rhapsody.jpg`,
    src: ``,
  }]
};

it(`Home Page correctly renders`, () => {
  const {films} = mock;
  const tree = renderer
    .create(<ListOfFilms
      films={films}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
