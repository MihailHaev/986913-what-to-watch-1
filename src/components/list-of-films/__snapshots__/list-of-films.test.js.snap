// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page correctly renders 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <video
      height="175"
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      src=""
      width="280"
    />
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <video
      height="175"
      muted={true}
      poster="img/bohemian-rhapsody.jpg"
      src=""
      width="280"
    />
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>,
]
`;
